src/index.tsx

```diff
@@ -8,7 +8,7 @@ export interface UseSuperclusterArgument<P, C> {
   points: Array<Supercluster.PointFeature<P>>;
   bounds?: BBox;
   zoom: number;
-  options?: Supercluster.Options<P, C>;
+  options?: Supercluster.Options<P, C> & { disableRefresh?: boolean };
```

Comment:
Sorry! I didn't explain well... this is what I was hoping for. Rather than changing what the `options` are, which are directly passed to Supercluster, I was hoping for a distinct `disableRefresh` prop.

```suggestion
  options?: Supercluster.Options<P, C>;
  disableRefresh?: boolean;
```